docs/monitoring/metrics.md

```diff
@@ -646,6 +646,7 @@ metrics.reporter.grph.factory.class: org.apache.flink.metrics.graphite.GraphiteR
 metrics.reporter.grph.host: localhost
 metrics.reporter.grph.port: 2003
 metrics.reporter.grph.protocol: TCP
+metrics.reporter.grph.interval: 60 SECONDS
```

Comment:
Is it not too long for most cases? Maybe `5-10 sec`?

Reply:
it is just an example.